website/docs/support/known-issues.md

```diff
@@ -35,3 +35,5 @@ Below is a list of known Corso issues and limitations:
 * Groups and Teams support is available in an early-access status, and may be subject to breaking changes.
 
 * Restoring the data into a different Group from the one it was backed up from isn't currently supported
+
+* CLI completions can't autocomplete multiple values for flags
```

Comment:
Could you please explain what this means?

Reply:
For example some of our flags like `--mailbox` can take multiple values as a comma separated list. We currently cannot handle that in the autocompletion engine.

Reply:
Thanks!